### ISSUE TITLE
@supports bug

### ISSUE THREAD

**Issue by thomashigginbotham** [NV/CSSOM#93]
While running cssom on Twitter Bootstrap's CSS, I received an error and was able to narrow it down to the use of @supports. Here's some sample code that will cause the error.

```css
@supports (transform-style: preserve-3d) {
  .foo {
    transform: translate3d(0, 0, 0);
  }
}
```
Error: `Unexpected }`

**Comment by thomashigginbotham** [comment]
I created a pull request that fixes this issue.

**Comment by NV** [comment]
Fixed in #96.
